test/structure-data-testing-tool.js

```diff
@@ -0,0 +1,87 @@
+const assert = require("assert");
+const { testSchema } = require("./utils");
+
+const showResult = (result) => {
+  if (result) {
+    console.log(
+      `Passed: ${result.passed.length},`,
+      `Failed: ${result.failed.length},`,
+      `Warnings: ${result.warnings.length}`
+    );
+    console.log(`Schemas found: ${result.schemas.join(",")}`);
+    // Loop over validation errors
+    if (result.failed.length > 0) {
+      console.log(
+        "⚠️  Errors:\n",
+        result.failed.map((test) => test)
+      );
+    }
+  }
+};
+
+describe("Structure data tool testing", () => {
+  const homePageSchema = ["Organization", "Website", "BreadcrumbList"];
+  it("Home page", async () => {
+    const url = "https://malibu-advanced-web.quintype.io";
+    const result = await testSchema(url, homePageSchema);
+    showResult(result);
+    assert.equal(0, result.failed.length);
+  }).timeout(5000);
+  it("Text story page", async () => {
+    const url =
+      "https://malibu-advanced-web.quintype.io/technology/automobiles/an-startling-fusion-of-technology-comfort-and-driver-experience";
+    const schemas = ["NewsArticle", "BreadcrumbList"];
+    const result = await testSchema(url, schemas);
+    showResult(result);
+    assert.equal(0, result.failed.length);
+  }).timeout(5000);
+  it("Photo story page", async () => {
+    const url = "https://malibu-advanced-web.quintype.io/entertainment/government-of-goa-official-portal";
+    const schemas = ["NewsArticle", "BreadcrumbList"];
```

Comment:
what about article, website and organization schema?

Reply:
that are not mandatory for story pages. that can add it through customization